### ISSUE TITLE
QUnit: Add assertContains()

### ISSUE THREAD

**Issue by geckon** [qorelanguage/qore#3374]
It would be nice to have something like `assertContains()` and `assertNContains()` or maybe `assertIn()` and `assertNIn()` or something like that to check that a list contains a value (or doesn't).

Maybe it could also work for hashes somehow.

**Comment by omusil24** [comment]
```
assertTrue(list.contains("abc"));
assertFalse(h.hasKey("abc"));
```
Not enough?

**Comment by geckon** [comment]
Good point. I guess these might be enough. I still think that it'd be nice to have asserts for that but it's probably not a priority.
